### ISSUE TITLE
Issues in man pages

### ISSUE THREAD

**Issue by hmartink** [systemd/systemd#31482]
### systemd version the issue has been seen with

255

### Used distribution

Archlinux, Debian Unstable, Fedora 40

### Linux kernel version used

n/a

### CPU architectures issue was seen on

x86_64

### Expected behaviour you didn't see

Dear systemd maintainer,
the manpage-l10n project maintains a large number of translations of
man pages both from a large variety of sources (including Systemd) as
well for a large variety of target languages.

During their work translators notice different possible issues in the
original (english) man pages. Sometimes this is a straightforward
typo, sometimes a hard to read sentence, sometimes this is a
convention not held up and sometimes we simply do not understand the
original.

We use several distributions as sources and update regularly (at
least every 2 month). This means we are fairly recent (some
distributions like archlinux also update frequently) but might miss
the latest upstream version once in a while, so the error might be
already fixed. We apologize and ask you to close the issue immediately
if this should be the case, but given the huge volume of projects and
the very limited number of volunteers we are not able to double check
each and every issue.

Secondly we translators see the manpages in the neutral po format,
i.e. converted and harmonized, but not the original source (be it man,
groff, xml or other). So we cannot provide a true patch (where
possible), but only an approximation which you need to convert into
your source format.

Finally the issues I'm reporting have accumulated over time and are
not always discovered by me, so sometimes my description of the
problem my be a bit limited - do not hesitate to ask so we can clarify
them.

I'm now reporting the errors for your project. If future reports
should use another channel, please let me know.

### Unexpected behaviour you saw
```
Man page: coredump.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: daemon.7
Issue:    Only one fullstop at the end of the sentence needed.

"If the service opens sockets or other files on it own, and those file "
"descriptors shall survive a restart, the daemon should store them in the "
"service manager via B<sd_notify>(3)  with I<FDSTORE=1>\\&.\\&."
--
Man page: homed.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: iocost.conf.5
Issue: defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: journalctl.1
Issue:    Colon at the end?

"The following commands are understood\\&. If none is specified the default "
"is to display journal records\\&."
--
Man page: journalctl.1
Issue:    \"months\", \"weeks\"  → \"weeks\", \"months\"

"B<--vacuum-time=> removes archived journal files older than the specified "
"timespan\\&. Accepts the usual \"s\" (default), \"m\", \"h\", \"days\", "
"\"months\", \"weeks\" and \"years\" suffixes, see B<systemd.time>(7)  for "
"details\\&."
--
Man page: journald.conf.5
Issue: defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: journal-remote.conf.5
Issue: defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: journal-upload.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: kernel-install.8
Issue:    devicetree → device tree   (as in previous paragraph?)

"The devicetree file contains a path, and this path specifies a location "
"relative to the kernel install tree\\&. A set of locations is checked, "
"including in particular /usr/lib/modules/I<KERNEL_VERSION>/dtb/, which is "
"the recommended location to place the dtb files under\\&. For example, with "
"\"broadcom/bcm2711-rpi-4-b\\&.dtb\" in the devicetree file, the device tree "
"blob for the Raspberry Pi 4 Model B would be installed, and the actual file "
"would be /usr/lib/modules/I<KERNEL_VERSION>/dtb/broadcom/bcm2711-rpi-4-b\\&."
"dtb\\&."
--
Man page: logind.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: mkosi.1
Issue:    Many paragraphs finish with an (unnecessary) final \\fR

--
Man page: mkosi.1
Issue:    \\f[V]systemd-nspawn\\f[R] → \\f[V]systemd-nspawn(1)\\f[R]

"This builds the image if it is not built yet, and then invokes \\f[V]systemd-"
"nspawn\\f[R] to acquire an interactive shell prompt in it.  An optional "
"command line may be specified after the \\f[V]shell\\f[R] verb, to be "
"invoked in place of the shell in the container.  Use \\f[V]-f\\f[R] in order "
"to rebuild the image unconditionally before acquiring the shell, see below.  "
"This command must be executed as \\f[V]root\\f[R].\\fR"

"Similar to \\f[V]shell\\f[R], but boots the image using \\f[V]systemd-"
"nspawn\\f[R].  An optional command line may be specified after the "
"\\f[V]boot\\f[R] verb, which can contain extra nspawn options as well as "
"arguments which are passed as the \\f[I]kernel command line\\f[R] to the "
"init system in the image.\\fR"
--
Man page: mkosi.1
Issue:    cpio → \\f[V]cpio(5)\\f[R]

"Similar to \\f[V]boot\\f[R], but uses \\f[V]qemu\\f[R] to boot up the image, "
"i.e.\\ instead of container virtualization virtual machine virtualization is "
"used.  This verb is only supported for disk images that contain a boot "
"loader and cpio images in which a kernel was installed.  For cpio images a "
"kernel can also be provided by passing the \\f[V]-kernel\\f[R] qemu argument "
"to the \\f[V]qemu\\f[R] verb.  Any arguments specified after the "
"\\f[V]qemu\\f[R] verb are appended to the \\f[V]qemu\\f[R] invocation.\\fR"
--
Man page: mkosi.1
Issue:    \\f[V]ssh\\f[R] → \\f[V]ssh(1)\\f[R]

"When the image is built with the \\f[V]Ssh=yes\\f[R] option, this command "
"connects to a booted virtual machine (\\f[V]qemu\\f[R]) via SSH.  Make sure "
"to run \\f[V]mkosi ssh\\f[R] with the same config as \\f[V]mkosi build\\f[R] "
"so that it has the necessary information available to connect to the running "
"virtual machine via SSH.  Specifically, the SSH private key from the "
"\\f[V]SshKey=\\f[R] setting is used to connect to the virtual machine.  Use "
"\\f[V]mkosi genkey\\f[R] to automatically generate a key and certificate "
"that will be picked up by mkosi.  Any arguments passed after the "
"\\f[V]ssh\\f[R] verb are passed as arguments to the \\f[V]ssh\\f[R] "
"invocation.  To connect to a container, use \\f[V]machinectl login\\f[R] or "
"\\f[V]machinectl shell\\f[R].\\fR"
--
Man page: mkosi.1
Issue:    \\f[V]journalctl\\f[R] → \\f[V]journalctl(1)\\f[R]

"Uses \\f[V]journalctl\\f[R] to inspect the journal inside the image.  Any "
"arguments specified after the \\f[V]journalctl\\f[R] verb are appended to "
"the \\f[V]journalctl\\f[R] invocation.\\fR"
--
Man page: mkosi.1
Issue:    \\f[V]coredumpctl\\f[R] → \\f[V]coredumpctl(1)\\f[R]

"Uses \\f[V]coredumpctl\\f[R] to look for coredumps inside the image.  Any "
"arguments specified after the \\f[V]coredumpctl\\f[R] verb are appended to "
"the \\f[V]coredumpctl\\f[R] invocation.\\fR"
--
Man page: mkosi.1
Issue:    mkosi → \\f[V]mkosi\\f[R]

"Show mkosi\\[cq]s documentation.  By default this verb will try several ways "
"to output the documentation, but a specific option can be chosen with the "
"\\f[V]--doc-format\\f[R] option.  Distro packagers are encouraged to add a "
"file \\f[V]mkosi.1\\f[R] into the \\f[V]mkosi/resources\\f[R] directory of "
"the Python package, if it is missing, as well as to install it in the "
"appropriate search path for man pages.  The man page can be generated from "
"the markdown file \\f[V]mkosi/resources/mkosi.md\\f[R] e.g via \\f[V]pandoc -"
"t man -s -o mkosi.1 mkosi.md\\f[R].\\fR"

"When executing a command in the image fails, mkosi will start an interactive "
"shell in the image allowing further debugging."

"Common name to be used when generating keys via mkosi\\[cq]s "
"\\f[V]genkey\\f[R] command.  Defaults to \\f[V]mkosi of %u\\f[R], where "
"\\f[V]%u\\f[R] expands to the username of the user invoking mkosi.\\fR"

"Number of days that the keys should remain valid when generating keys via "
"mkosi\\[cq]s \\f[V]genkey\\f[R] command.  Defaults to two years (730 days)."
"\\fR"

"The output format may also be set to \\f[I]none\\f[R] to have mkosi produce "
"no image at all.  This can be useful if you only want to use the image to "
"produce another output in the build scripts (e.g.\\ build an rpm).\\fR"
--
Man page: mkosi.1
Issue 1:  mkosi → \\f[V]mkosi\\f[R]
Issue 2:  \\f[V]pandoc\\f[R] → \\f[V]pandoc(1)\\f[R]

"The format to show the documentation in.  Supports the values "
"\\f[V]markdown\\f[R], \\f[V]man\\f[R], \\f[V]pandoc\\f[R], "
"\\f[V]system\\f[R] and \\f[V]auto\\f[R].  In the case of "
"\\f[V]markdown\\f[R] the documentation is shown in the original Markdown "
"format.  \\f[V]man\\f[R] shows the documentation in man page format, if it "
"is available.  \\f[V]pandoc\\f[R] will generate the man page format on the "
"fly, if \\f[V]pandoc\\f[R] is available.  \\f[V]system\\f[R] will show the "
"system-wide man page for mkosi, which may or may not correspond to the "
"version you are using, depending on how you installed mkosi.  "
"\\f[V]auto\\f[R], which is the default, will try all methods in the order "
"\\f[V]man\\f[R], \\f[V]pandoc\\f[R], \\f[V]markdown\\f[R], "
"\\f[V]system\\f[R].\\fR"
--
Man page: mkosi.1
Issue:    systemd-repart → \\f[V]systemd-repart(8)\\f[R]

"Raw \\f[I]GPT\\f[R] disk image, created using systemd-repart "
"(\\f[I]disk\\f[R])\\fR"
--
Man page: mkosi.1
Issue 1:  mkosi → \\f[V]mkosi\\f[R]
Issue 2:  XFS → \\f[V]xfs(5)\\f[R]
Issue 3:  btrfs → \\f[V]btrfs(5)\\f[R]

"It is highly recommended to run \\f[V]mkosi\\f[R] on a file system that "
"supports reflinks such as XFS and btrfs and to keep all related directories "
"on the same file system.  This allows mkosi to create images very quickly by "
"using reflinks to perform copying via copy-on-write operations.\\fR"
--
Man page: networkctl.1
Issue:    B<systemd-networkd> → B<systemd-networkd>(8)

"B<systemd-networkd> is not handling the link\\&."

"B<systemd-networkd> failed to configure the link\\&."

"The link is gone, but has not yet been dropped by B<systemd-networkd>\\&."
--
Man page: networkd.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: oomd.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: os-release.5
Issue:    As in systemd-portabled.service(8): Portable Services → Portable Services Documentation?

"Takes a space-separated list of one or more valid prefix match strings for "
"the \\m[blue]B<Portable Services>\\m[]\\&\\s-2\\u[3]\\d\\s+2 logic\\&. This "
"field serves two purposes: it is informational, identifying portable service "
"images as such (and thus allowing them to be distinguished from other OS "
"images, such as bootable system images)\\&. It is also used when a portable "
"service image is attached: the specified or implied portable service prefix "
"is checked against the list specified here, to enforce restrictions how "
"images may be attached to a system\\&."

"Portable Services"
--
Man page: portablectl.1
Issue:    and\\&. → and

"Add an additional image I<PATH> as an overlay on top of I<IMAGE> when "
"attaching/detaching\\&. This argument can be specified multiple times, in "
"which case the order in which images are laid down follows the rules "
"specified in B<systemd.exec>(5)  for the I<ExtensionImages=> directive and "
"for the B<systemd-sysext>(8)  and\\&.  B<systemd-confext>(8)  tools\\&. The "
"images must contain an extension-release file with metadata that matches "
"what is defined in the os-release of I<IMAGE>\\&. See: B<os-release>(5)\\&. "
"Images can be block images, btrfs subvolumes or directories\\&. For more "
"information on portable services with extensions, see the \"Extension "
"Images\" paragraph on \\m[blue]B<Portable Services "
"Documentation>\\m[]\\&\\s-2\\u[1]\\d\\s+2\\&."
--
Man page: poweroff.8
Issue:    B<halt> → B<halt>(8)

"Note that on many SysV systems B<halt> used to be synonymous to B<poweroff>, "
"i\\&.e\\&. both commands would equally result in powering the machine "
"off\\&. systemd is more accurate here, and B<halt> results in halting the "
"machine only (leaving power on), and B<poweroff> is required to actually "
"power it off\\&."
--
Man page: pstore.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: repart.d.5
Issue:    btrfs → B<btrfs>(5)

"Note that this option only takes effect if the target filesystem supports "
"subvolumes, such as \"btrfs\"\\&."
--
Man page: repart.d.5
Issue:    mkfs\\&.btrfs → B<mkfs.btrfs>(8)

"Note that due to limitations of \"mkfs\\&.btrfs\", this option is only "
"supported when running with B<--offline=no>\\&."
--
Man page: repart.d.5
Issue:    systemd-repart → B<systemd-repart>(8)

"Configures the data block size of the generated verity hash partition\\&. "
"Must be between 512 and 4096 bytes and must be a power of 2\\&. Defaults to "
"the sector size if configured explicitly, or the underlying block device "
"sector size, or 4K if systemd-repart is not operating on a block device\\&."

"Configures the hash block size of the generated verity hash partition\\&. "
"Must be between 512 and 4096 bytes and must be a power of 2\\&. Defaults to "
"the sector size if configured explicitly, or the underlying block device "
"sector size, or 4K if systemd-repart is not operating on a block device\\&."
--
Man page: resolved.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: resolved.conf.5
Issue:    B<systemd-resolved> → B<systemd-resolved>(8)

"B<systemd-resolved> always attempts to reach the upstream DNS servers first, "
"before providing the client application with any stale data\\&. If this "
"feature is enabled, cache will not be flushed when changing servers\\&."
--
Man page: systemctl.1
Issue:   First sentence broken? can still be → is

"If a unit that gets stopped can still be triggered by other units, a warning "
"containing the names of the triggering units is shown\\&.  B<--no-warn> can "
"be used to suppress the warning\\&."
--
Man page: systemd-boot.7
Issue:  Is "option" really part of the quoted string?

"A \"Reboot Into Firmware Interface option\", if supported by the UEFI "
"firmware\\&."
--
Man page: systemd-bsod.service.8
Issue:   No fullstop for NAME line

"systemd-bsod.service, systemd-bsod - Displays boot-time emergency log "
"message in full screen\\&."
--
Man page: systemd-bsod.service.8
Issue:    systemd-bsod\\&.service → B<systemd-bsod\\&.service>

"systemd-bsod\\&.service is used to display a blue screen which contains a "
"message relating to a boot failure, including a QR code which can be scanned "
"to get helpful information about the failure\\&."
--
Man page: systemd-cryptsetup.8
Issue 1:  systemd-cryptsetup → B<systemd-cryptsetup>
Issue 2:  systemd-cryptsetup@\\&.service → B<systemd-cryptsetup@\\&.service>(8)
Issue 3:  In SYNPOSIS SOURCE-DEVICE, here SOURCEDEVICE
Issue 4:  CRYPTTAB-OPTIONS does not appear in SYNPOSIS, but CONFIG is used there

"systemd-cryptsetup is used to set up (with B<attach>) and tear down (with "
"B<detach>) access to an encrypted block device\\&. It is primarily used via "
"systemd-cryptsetup@\\&.service during early boot, but may also be be called "
"manually\\&. The positional arguments I<VOLUME>, I<SOURCEDEVICE>, I<KEY-"
"FILE>, and I<CRYPTTAB-OPTIONS> have the same meaning as the fields in "
"B<crypttab>(5)\\&."
--
Man page: systemd-cryptsetup.8
Issue:    systemd-cryptsetup@\\&.service → B<systemd-cryptsetup@\\&.service>(8)

"systemd-cryptsetup@\\&.service is a service responsible for providing access "
"to encrypted block devices\\&. It is instantiated for each device that "
"requires decryption\\&."

"systemd-cryptsetup@\\&.service instances are part of the system-"
"systemd\\ex2dcryptsetup\\&.slice slice, which is destroyed only very late in "
"the shutdown procedure\\&. This allows the encrypted devices to remain up "
"until filesystems have been unmounted\\&."

"In order to unlock a volume a password or binary key is required\\&.  "
"systemd-cryptsetup@\\&.service tries to acquire a suitable password or "
"binary key via the following mechanisms, tried in order:"
--
Man page: systemd-fsck@.service.8.po
Issue:   B<fsck> → B<fsck>(8)

"systemd-fsck-root\\&.service and systemd-fsck-usr\\&.service will activate "
"reboot\\&.target if B<fsck> returns the \"System should reboot\" condition, "
"or emergency\\&.target if B<fsck> returns the \"Filesystem errors left "
"uncorrected\" condition\\&."

"systemd-fsck@\\&.service will fail if B<fsck> returns with either \"System "
"should reboot\" or \"Filesystem errors left uncorrected\" conditions\\&. For "
"filesystems listed in /etc/fstab without \"nofail\" or \"noauto\" options, "
"\"local-fs\\&.target\" will then activate emergency\\&.target\\&."
--
Man page: systemd-hibernate-resume.service.8
Issue:    B<systemd-hibernate-resume> → B<systemd-hibernate-resume>(8)

"B<systemd-hibernate-resume> only supports the in-kernel hibernation "
"implementation, see \\m[blue]B<Swap suspend>\\m[]\\&\\s-2\\u[1]\\d\\s+2\\&. "
"Internally, it works by writing the major:minor of specified device node to /"
"sys/power/resume, along with the offset in memory pages (/sys/power/"
"resume_offset) if supported\\&."
--
Man page: systemd.index.7
Issue:    Remove full stop

"B<systemd-bsod>(8)  \\(em Displays boot-time emergency log message in full "
"screen\\&."

"B<systemd-vmspawn>(1)  \\(em Spawn an OS in a virtual machine\\&."
--
Man page: systemd.mount.5
Issue:    What does "If dosn't" in the last sentence relates to?        If → It ?

"Takes a boolean argument\\&. If true or no argument, a I<BindsTo=> "
"dependency on the backing device is set\\&. If false, the mount unit is not "
"stopped no matter whether the backing device is still present\\&. This is "
"useful when the file system is backed by volume managers\\&. If not set, and "
"the mount comes from unit fragments, i\\&.e\\&. generated from /etc/fstab by "
"B<systemd-fstab-generator>(8)  or loaded from a manually configured mount "
"unit, a combination of I<Requires=> and I<StopPropagatedFrom=> dependencies "
"is set on the backing device\\&. If doesn\\*(Aqt, only I<Requires=> is "
"used\\&."
--
Man page: systemd.network.5
Issue:    B<systemd-networkd> → B<systemd-networkd>(8)

"Note that any network interfaces that have the I<ID_NET_MANAGED_BY=> udev "
"property set will never be matched by any \\&.network files \\(en unless the "
"property\\*(Aqs value is the string \"io\\&.systemd\\&.Network\" \\(en even "
"if the [Match] section would otherwise match\\&. This may be used to exclude "
"specific network interfaces from B<systemd-networkd>\\*(Aqs management, "
"while keeping the [Match] section generic\\&. The I<ID_NET_MANAGED_BY=> "
"property thus declares intended I<ownership> of the device, and permits "
"ensuring that concurrent network management implementations do not compete "
"for management of specific devices\\&."
--
Man page: systemd.network.5
Issue 1:  \"strict\", mode each → \"strict\" mode, each
Issue 2:  \"loose\", mode each → \"loose\" mode, each

"Configure IPv4 Reverse Path Filtering\\&. If enabled, when an IPv4 packet is "
"received, the machine will first check whether the I<source> of the packet "
"would be routed through the interface it came in\\&. If there is no route to "
"the source on that interface, the machine will drop the packet\\&. Takes one "
"of \"no\", \"strict\", or \"loose\"\\&. When \"no\", no source validation "
"will be done\\&. When \"strict\", mode each incoming packet is tested "
"against the FIB and if the incoming interface is not the best reverse path, "
"the packet check will fail\\&. By default failed packets are discarded\\&. "
"When \"loose\", mode each incoming packet\\*(Aqs source address is tested "
"against the FIB\\&. The packet is dropped only if the source address is not "
"reachable via any interface on that router\\&. See \\m[blue]B<RFC "
"3704>\\m[]\\&\\s-2\\u[9]\\d\\s+2\\&. When unset, the kernel\\*(Aqs default "
"will be used\\&."
--
Man page: systemd.network.5
Issue:    B<systemd-networkd> → B<systemd-networkd>(8)

"When an interface is configured with IP addresses, the addresses, subnetwork "
"masks or interface index will be appended to the NFT sets\\&. The "
"information will be removed when the interface is deconfigured\\&.  "
"B<systemd-networkd> only inserts elements to (or removes from) the sets, so "
"the related NFT rules, tables and sets must be prepared elsewhere in "
"advance\\&. Failures to manage the sets will be ignored\\&."
--
Man page: systemd.network.5
Issue:    in the above → above

"Note that this filters only DHCP offers, so the filtering may not work when "
"I<RapidCommit=> is enabled\\&. See also I<RapidCommit=> in the above\\&."
--
Man page: systemd.network.5
Issue:    following\\&. → following:

"are equivalent to the following\\&."
--
Man page: systemd.network.5
Issue 1:  An integer the default → An integer\\&. The default
Issue 2:  The default unit is seconds, but the range is given in msec?

"Takes a timespan\\&. Configures the retransmit time, used by clients to "
"retransmit Neighbor Solicitation messages on address resolution and the "
"Neighbor Unreachability Detection algorithm\\&. An integer the default unit "
"of seconds, in the range 0\\&...4294967295 msec\\&. Defaults to 0\\&."
--
Man page: systemd.network.5
Issue 1:  Agent Option → Agent option
Issue 2:  which indicates → indicate

"Takes a boolean\\&. Specifies that IPv6 router advertisements which "
"indicates to hosts that the router acts as a Home Agent and includes a Home "
"Agent Option\\&. Defaults to false\\&. See \\m[blue]B<RFC "
"6275>\\m[]\\&\\s-2\\u[10]\\d\\s+2 for further details\\&."
--
Man page: systemd.network.5
Issue 1:  An integer the → An integer\\&. The
Issue 2:  of seconds → is seconds

"Takes a timespan\\&. Specifies the lifetime of the Home Agent\\&. An integer "
"the default unit of seconds, in the range 1\\&...65535\\&. Defaults to the "
"value set to I<RouterLifetimeSec=>\\&."
--
Man page: systemd-nspawn.1
Issue:    !\\&. → !

"In order to embed binary data into the credential data for B<--set-"
"credential=>, use C-style escaping (i\\&.e\\&.  \"\\en\" to embed a newline, "
"or \"\\ex00\" to embed a B<NUL> byte)\\&. Note that the invoking shell might "
"already apply unescaping once, hence this might require double escaping!\\&."
--
Man page: systemd.pcrlock.5
Issue:   B<systemd-pcrlock>(1) → B<systemd-pcrlock>(8)

"*\\&.pcrlock files define expected TPM2 PCR measurements of components "
"involved in the boot process\\&.  B<systemd-pcrlock>(1)  uses such pcrlock "
"files to analyze and predict TPM2 PCR measurements\\&. The pcrlock files are "
"JSON arrays that follow a subset of the \\m[blue]B<TCG Common Event Log "
"Format (CEL-JSON)>\\m[]\\&\\s-2\\u[1]\\d\\s+2 specification\\&. Specifically "
"the \"recnum\", \"content\", and \"content_type\" record fields are not used "
"and ignored if present\\&. Each pcrlock file defines one set of expected, "
"ordered PCR measurements of a specific component of the boot\\&."

"Use the various B<lock-*> commands of B<systemd-pcrlock> to automatically "
"generate suitable pcrlock files for various types of resources\\&."

--
Man page: systemd.pcrlock.5
Issue:    400-secureboot-separator\\&.pcrlock\\&. → 400-secureboot-separator\\&.pcrlock (i.e. no final full stop)

"Firmware code measurements, as recorded to PCR 0 and 2, up to the separator "
"measurement (see 400-secureboot-separator\\&.pcrlock\\&.  below)\\&. May be "
"generated via B<systemd-pcrlock lock-firmware-code>\\&."

"Firmware configuration measurements, as recorded to PCR 1 and 3, up to the "
"separator measurement (see 400-secureboot-separator\\&.pcrlock\\&.  "
"below)\\&. May be generated via B<systemd-pcrlock lock-firmware-config>\\&."

"Firmware code measurements, as recorded to PCR 0 and 2, after the separator "
"measurement (see 400-secureboot-separator\\&.pcrlock\\&.  above)\\&. May be "
"generated via B<systemd-pcrlock lock-firmware-code>\\&."

"Firmware configuration measurements, as recorded to PCR 1 and 3, after the "
"separator measurement (see 400-secureboot-separator\\&.pcrlock\\&.  "
"above)\\&. May be generated via B<systemd-pcrlock lock-firmware-config>\\&."
--
Man page: systemd.pcrlock.5
Issue:    the UEFI → when the UEFI?

"The EFI action generated when B<ExitBootServices()> is generated, i\\&.e\\&. "
"the UEFI environment is left and the OS takes over\\&. Covers the PCR 5 "
"measurement\\&. Statically defined\\&."
--
Man page: systemd-pcrlock.8.
Issue:    The word "format" is already in the cited name, remove "format" at the end of the sentence?

"This reads the combined TPM2 event log and writes it to STDOUT in "
"\\m[blue]B<TCG Common Event Log Format (CEL-"
"JSON)>\\m[]\\&\\s-2\\u[1]\\d\\s+2 format\\&."
--
Man page: systemd-pcrlock.8.
Issue:    B<systemd-stub> → B<systemd-stub>(7)

"Generates/removes a \\&.pcrlock file based on a kernel initrd cpio "
"archive\\&. This is useful for predicting measurements the Linux kernel "
"makes to PCR 9 (\"kernel-initrd\")\\&. Do not use for B<systemd-stub> UKIs, "
"as the initrd is combined dynamically from various sources and hence does "
"not take a single input, like this command\\&."
--
Man page: systemd-pcrlock.8.
Issue:    On second occurence: pcrlock file → \\&.pcrlock file

"Generates/removes a \\&.pcrlock file based on raw binary data\\&. The data "
"is either read from the specified file or from STDIN (if none is "
"specified)\\&. This requires that B<--pcrs=> is specified\\&. The generated "
"pcrlock file is written to the file specified via B<--pcrlock=> or to STDOUT "
"(if none is specified)\\&."
--
Man page: systemd-pcrlock.8.
Issue:    Specifies to → Specifies the

"Specifies to NV index to store the policy in\\&. Honoured by B<make-"
"policy>\\&. If not specified the command will automatically pick a free NV "
"index\\&."
--
Man page: systemd-pcrlock.8.
Issue:    overrides → configures

"Takes a file system path as argument\\&. If specified overrides where to "
"write pcrlock policy metadata to\\&. If not specified defaults to /var/lib/"
"systemd/pcrlock\\&.json\\&."
--
Man page: systemd-pcrphase.service.8
Issue:    systemd-pcrphase\\&.service → B<systemd-pcrphase\\&.service>   (This is the name of this man page, hence bold)

"\"ready\" \\(em during later boot-up, after remote file systems have been "
"activated (i\\&.e\\&. after remote-fs\\&.target), but before users are "
"permitted to log in (i\\&.e\\&. before systemd-user-sessions\\&.service)\\&. "
"It acts as barrier between the time where unprivileged regular users are "
"still prohibited to log in and where they are allowed to log in\\&. (This "
"extension happens when the systemd-pcrphase\\&.service service is "
"started\\&.)"

"\"shutdown\" \\(em when the system shutdown begins\\&. It acts as barrier "
"between the time the system is fully up and running and where it is about to "
"shut down\\&. (This extension happens when the systemd-pcrphase\\&.service "
"service is stopped\\&.)"
--
Man page: systemd-portabled.service.8
Issue:    In systemd-portabled.service(8): Portable Services Documentation

"Portable Services"
--
Man page: systemd-poweroff.service.8
Issue:    systemd-poweroff\\&.service → B<systemd-poweroff.service>

"systemd-poweroff\\&.service is a system service that is pulled in by "
"poweroff\\&.target and is responsible for the actual system power-off "
"operation\\&. Similarly, systemd-halt\\&.service is pulled in by halt\\&."
"target, systemd-reboot\\&.service by reboot\\&.target and systemd-kexec\\&."
"service by kexec\\&.target to execute the respective actions\\&."
--
Man page: systemd-poweroff.service.8
Issue 1:  systemd-shutdown → B<systemd-shutdown>
Issue 2:  have been detached → have been unmounted (?)

"Shortly before executing the actual system power-off/halt/reboot/kexec "
"systemd-shutdown will run all executables in /usr/lib/systemd/system-"
"shutdown/ and pass one arguments to them: either \"poweroff\", \"halt\", "
"\"reboot\", or \"kexec\", depending on the chosen action\\&. All executables "
"in this directory are executed in parallel, and execution of the action is "
"not continued before all executables finished\\&. Note that these "
"executables are run I<after> all services have been shut down, and after "
"most mounts have been detached (the root file system as well as /run/ and "
"various API file systems are still around though)\\&. This means any "
"programs dropped into this directory must be prepared to run in such a "
"limited execution environment and not rely on external services or "
"hierarchies such as /var/ to be around (or writable)\\&."
--
Man page: systemd-poweroff.service.8
Issue:    systemd-poweroff\\&.service → B<systemd-poweroff\\&.service>

"Note that systemd-poweroff\\&.service (and the related units) should never "
"be executed directly\\&. Instead, trigger system shutdown with a command "
"such as \"systemctl poweroff\"\\&."
--
Man page: systemd-repart.8
Issue:    update\\&. → update:

"The following creates a configuration extension DDI (confext) for an /etc/"
"motd update\\&."
--
Man page: systemd-repart.8
Issue:    In systemd-portabled.service(8): Portable Services Documentation

"portable service"
--
Man page: systemd-resolved.service.8
Issue:    systemd-resolved\\&.service → B<systemd-resolved\\&.service>

"The nss-dns resolver maintains little state between subsequent DNS queries, "
"and for each query always talks to the first listed DNS server from /etc/"
"resolv\\&.conf first, and on failure continues with the next until reaching "
"the end of the list which is when the query fails\\&. The resolver in "
"systemd-resolved\\&.service however maintains state, and will continuously "
"talk to the same server for all queries on a particular lookup scope until "
"some form of error is seen at which point it switches to the next, and then "
"continuously stays with it for all queries on the scope until the next "
"failure, and so on, eventually returning to the first configured server\\&. "
"This is done to optimize lookup times, in particular given that the resolver "
"typically must first probe server feature sets when talking to a server, "
"which is time consuming\\&. This different behaviour implies that listed DNS "
"servers per lookup scope must be equivalent in the zones they serve, so that "
"sending a query to one of them will yield the same results as sending it to "
"another configured DNS server\\&."
--
Man page: systemd.scope.5
Issue:    There is a man page, hence the reference: systemd-oomd\\&.service → B<systemd-oomd\\&.service>(8) (2nd occurence after Note) // We don't use bold for service file names.

"Configure the out-of-memory (OOM) killing policy for the kernel and the "
"userspace OOM killer B<systemd-oomd.service>(8)\\&. On Linux, when memory "
"becomes scarce to the point that the kernel has trouble allocating memory "
"for itself, it might decide to kill a running process in order to free up "
"memory and reduce memory pressure\\&. Note that systemd-oomd\\&.service is a "
"more flexible solution that aims to prevent out-of-memory situations for the "
"userspace too, not just the kernel, by attempting to terminate services "
"earlier, before the kernel would have to act\\&."
--
Man page: systemd.service.5
Issue:    There is a man page, hence the reference: systemd-oomd\\&.service → B<systemd-oomd\\&.service>(8) (2nd occurence after Note) // We don't use bold for service file names.

"Configure the out-of-memory (OOM) killing policy for the kernel and the "
"userspace OOM killer B<systemd-oomd.service>(8)\\&. On Linux, when memory "
"becomes scarce to the point that the kernel has trouble allocating memory "
"for itself, it might decide to kill a running process in order to free up "
"memory and reduce memory pressure\\&. Note that systemd-oomd\\&.service is a "
"more flexible solution that aims to prevent out-of-memory situations for the "
"userspace too, not just the kernel, by attempting to terminate services "
"earlier, before the kernel would have to act\\&."
--
Man page: systemd-sleep.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: systemd-soft-reboot.service.8
Issue:    ordeering → ordering

"Service processes may continue to run over the transition, past soft-reboot "
"and into the next session, if they are placed in services that remain active "
"until the very end of shutdown (which again is achieved via "
"I<DefaultDependencies=no>)\\&. They must also be set up to avoid being "
"killed by the aforementioned B<SIGTERM> and B<SIGKILL> via "
"I<SurviveFinalKillSignal=yes>, and also be configured to avoid being stopped "
"on isolate via I<IgnoreOnIsolate=yes>\\&. They also have to be configured to "
"be stopped on normal shutdown, reboot and maintenance mode\\&. Finally, they "
"have to be ordered after B<basic\\&.target> to ensure correct ordeering on "
"boot\\&. Note that in case any new or custom units are used to isolate to, "
"or that implement an equivalent shutdown functionality, they will also have "
"to be configured manually for correct ordering and conflicting\\&. For "
"example:"
--
Man page: systemd-soft-reboot.service.8
Issue:    systemd-soft-reboot\\&.service → B<systemd-soft-reboot.service>

"Note that systemd-soft-reboot\\&.service (and related units) should never be "
"executed directly\\&. Instead, trigger system shutdown with a command such "
"as \"systemctl soft-reboot\"\\&."
--
Man page: systemd-soft-reboot.service.8
Issue:    In systemd-portabled.service(8): Portable Services Documentation

"Portable Services"
--
Man page: systemd-storagetm.service.8
Issue:    systemd-storagetm\\&.service → B<systemd-storagetm\\&.service>

"systemd-storagetm\\&.service is a service that exposes all local block "
"devices as NVMe-TCP mass storage devices\\&. Its primary use-case is to be "
"invoked by the storage-target-mode\\&.target unit that can be booted into\\&."
--
Man page: systemd-storagetm.service.8
Issue:    the the → the

"Takes a string\\&. If specified configures the NVMe Qualified Name to use "
"for the exposed NVMe-TCP mass storage devices\\&. The NQN should follow the "
"syntax described in \\m[blue]B<NVM Express Base Specification "
"2\\&.0c>\\m[]\\&\\s-2\\u[1]\\d\\s+2, section 4\\&.5 \"NVMe Qualified "
"Names\"\\&. Note that the NQN specified here will be suffixed with a dot and "
"the the block device name before it is exposed on the NVMe target\\&. If not "
"specified defaults to \"nqn\\&.2023-10\\&.io\\&.systemd:storagetm\\&."
"I<ID>\", where ID is replaced by a 128bit ID derived from B<machine-"
"id>(5)\\&."
--
Man page: systemd-sysext.8
Issue 1:  Note that the → Note that
Issue 2:  earlier: → earlier

"A simple mechanism for version compatibility is enforced: a system extension "
"image must carry a /usr/lib/extension-release\\&.d/extension-release\\&."
"I<NAME> file, which must match its image name, that is compared with the "
"host os-release file: the contained I<ID=> fields have to match unless "
"\"_any\" is set for the extension\\&. If the extension I<ID=> is not "
"\"_any\", the I<SYSEXT_LEVEL=> field (if defined) has to match\\&. If the "
"latter is not defined, the I<VERSION_ID=> field has to match instead\\&. If "
"the extension defines the I<ARCHITECTURE=> field and the value is not "
"\"_any\" it has to match the kernel\\*(Aqs architecture reported by "
"B<uname>(2)  but the used architecture identifiers are the same as for "
"I<ConditionArchitecture=> described in B<systemd.unit>(5)\\&.  "
"I<EXTENSION_RELOAD_MANAGER=> can be set to 1 if the extension requires a "
"service manager reload after application of the extension\\&. Note that the "
"for the reasons mentioned earlier: \\m[blue]B<Portable "
"Services>\\m[]\\&\\s-2\\u[2]\\d\\s+2 remain the recommended way to ship "
"system services\\&. System extensions should not ship a /usr/lib/os-release "
"file (as that would be merged into the host /usr/ tree, overriding the host "
"OS version data, which is not desirable)\\&. The extension-release file "
"follows the same format and semantics, and carries the same content, as the "
"os-release file of the OS, but it describes the resources carried in the "
"extension image\\&."
--
Man page: systemd-sysext.8
Issue:    In systemd-portabled.service(8): Portable Services Documentation

"Portable Services"
--
Man page: systemd-system.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: systemd.time.7
Issue:    What does &c after Tokyo stand for?

"The timezone defaults to the current timezone if not specified "
"explicitly\\&. It may be given after a space, like above, in which case it "
"can be: \"UTC\", an entry in the installed IANA timezone database (\"CET\", "
"\"Asia/Tokyo\", &c\\&.; complete list obtainable with \"timedatectl list-"
"timezones\" (see B<timedatectl>(1))), or \"\\(+-I<05>\", \"\\(+-"
"I<05>I<30>\", \"\\(+-I<05>:I<30>\", \"Z\"\\&."
--
Man page: systemd-tpm2-setup.service.8
Issue:    Singular or plural? 1st sentence: "The services", 2nd sentence: "It will"

"The services will store the public key of the SRK key pair in a PEM file in /"
"run/systemd/tpm2-srk-public-key\\&.pem and /var/lib/systemd/tpm2-srk-public-"
"key\\&.pem\\&. It will also store it in TPM2B_PUBLIC format in /run/systemd/"
"tpm2-srk-public-key\\&.tpm2_public and /var/lib/systemd/tpm2-srk-public-"
"key\\&.tpm2b_public\\&."
--
Man page: systemd.unit.5
Issue:    then this the → then this

"For units that start processes (such as service units), lists one or more "
"other units whose network and/or temporary file namespace to join\\&. If "
"this is specified on a unit (say, a\\&.service has I<JoinsNamespaceOf=b\\&."
"service>), then the inverse dependency (I<JoinsNamespaceOf=a\\&.service> for "
"b\\&.service) is implied\\&. This only applies to unit types which support "
"the I<PrivateNetwork=>, I<NetworkNamespacePath=>, I<PrivateIPC=>, "
"I<IPCNamespacePath=>, and I<PrivateTmp=> directives (see B<systemd.exec>(5)  "
"for details)\\&. If a unit that has this setting set is started, its "
"processes will see the same /tmp/, /var/tmp/, IPC namespace and network "
"namespace as one listed unit that is started\\&. If multiple listed units "
"are already started and these do not share their namespace, then it is not "
"defined which namespace is joined\\&. Note that this setting only has an "
"effect if I<PrivateNetwork=>/I<NetworkNamespacePath=>, I<PrivateIPC=>/"
"I<IPCNamespacePath=> and/or I<PrivateTmp=> is enabled for both the unit that "
"joins the namespace and the unit whose namespace is joined\\&."
--
Man page: systemd-vmspawn.1
Issue:    No full stop in NAME

msgid "systemd-vmspawn - Spawn an OS in a virtual machine\\&."
--
Man page: systemd-vmspawn.1
Issue:    systemd-vmspawn → B<systemd-vmspawn>

"Note: on Ubuntu/Debian derivatives systemd-vmspawn requires the user to be "
"in the \"kvm\" group to use the VSock options\\&."
--
Man page: systemd-vmspawn.1
Issue:    !\\&. → !

"In order to embed binary data into the credential data for B<--set-"
"credential=>, use C-style escaping (i\\&.e\\&.  \"\\en\" to embed a newline, "
"or \"\\ex00\" to embed a B<NUL> byte)\\&. Note that the invoking shell might "
"already apply unescaping once, hence this might require double escaping!\\&."
--
Man page: timesyncd.conf.5
Issue:    defined → defines

"When packages need to customize the configuration, they can install drop-ins "
"under /usr/\\&. Files in /etc/ are reserved for the local administrator, who "
"may use this logic to override the configuration files installed by vendor "
"packages\\&. Drop-ins have to be used to override package drop-ins, since "
"the main configuration file has lower precedence\\&. It is recommended to "
"prefix all filenames in those subdirectories with a two-digit number and a "
"dash, to simplify the ordering of the files\\&. This also defined a concept "
"of drop-in priority to allow distributions to ship drop-ins within a "
"specific range lower than the range used by users\\&. This should lower the "
"risk of package drop-ins overriding accidentally drop-ins defined by "
"users\\&."
--
Man page: tmpfiles.d.5
Issue:    B<systemd-tmpfiles> → B<systemd-tmpfiles>(8)

"By passing this line to QEMU, the public key of the current user will be "
"encoded in base64, added to a tmpfiles\\&.d line that tells B<systemd-"
"tmpfiles> to decode it into /root/\\&.ssh/authorized_keys, encode that line "
"itself in base64 and pass it as a Credential that will be picked up by "
"systemd from SMBIOS on boot\\&."
--
Man page: ukify.1
Issue:    last words: to the file named I<PATH> or to a file in the I<PATH>?

"For the B<inspect> verb, the second syntax is used\\&. The section I<NAME> "
"will be inspected (if found)\\&. If the second argument is \"text\", the "
"contents will be printed\\&. If the third argument is given, the contents "
"will be saved to file I<PATH>\\&."
--
Man page: varlinkctl.1
Issue:    in the formats → in one of the formats  (also in subsequent strings)

"Show brief information about the specified service, including vendor name "
"and list of implemented interfaces\\&. Expects a service address in the "
"formats described above\\&."
--
Man page: varlinkctl.1
Issue:    In SYNOPSIS it is PARAMETER not ARGUMENTS

"B<call> I<ADDRESS> I<METHOD> [I<ARGUMENTS>]"
--
Man page: veritytab.5
Issue:    The prase "For format" is unclear // Oh, that whole page uses strange telegraphic language. It should be reworded.

"Use forward error correction (FEC) to recover from corruption if hash "
"verification fails\\&. Use encoding data from the specified device\\&. The "
"fec device argument can be block device or file image\\&. For format, if fec "
"device path doesn\\*(Aqt exist, it will be created as file\\&. Note: block "
"sizes for data and hash devices must match\\&. Also, if the verity "
"data_device is encrypted the fec_device should be too\\&."
```
### Steps to reproduce the problem

Open man pages
